Addition of invalid character in webservicesdkpassword text

dakoer opened the pull request:
Support engineer determined that the character "&" is not allowed in the password text used for the webservicessdkpassword
Please refer to

dakoer committed:
Addition of invalid character in webservicesdkpassword text

Support engineer determined that the character "&" is not allowed in the password text used for the webservicessdkpassword
Please refer to

Justinha reviewed (changes requested):
@dakoer Thanks David! Can you add this as a second sentence to step 1, without Note? I see many readers skip over our notes, so I avoid them now.

dakoer committed:
Update howto-mfaserver-adfs-2012.md

Added password character update out of the note and into the account password step.
